stats/stats/src/charts/insert.rs

```diff
@@ -97,6 +97,11 @@ impl DateValue {
             self
         }
     }
+
+    pub fn can_be_partial(&self) -> bool {
```

Comment:
Nit: imo, `can_be_partial` does not represent what function does. My suggestion is to name it `is_partial`